tool/ood-gen/lib/changelog.ml

```diff
@@ -46,6 +46,44 @@ let all () =
   Utils.map_files decode "changelog/*/*.md"
   |> List.sort (fun a b -> String.compare b.slug a.slug)
 
+module ChangelogFeed = struct
+  let create_changelog_feed () =
+    let id = Uri.of_string "https://ocaml.org/changelog/feed.xml" in
+    let title : Syndic.Atom.title = Text "OCaml Changelog" in
+    let now = Ptime.of_float_s (Unix.gettimeofday ()) |> Option.get in
+    let cutoff_date =
+      Ptime.sub_span now (Ptime.Span.v (365, 0L)) |> Option.get
+    in
+
+    let entries =
+      all ()
+      |> List.map (fun (log : t) ->
+             let content = Syndic.Atom.Html (None, log.body_html) in
+             let id =
+               Uri.of_string ("https://ocaml.org/changelog/" ^ log.slug)
+             in
+             let authors = (Syndic.Atom.author "Ocaml.org", []) in
+             let updated =
+               Syndic.Date.of_rfc3339 (log.date ^ "T00:00:00-00:00")
```

Comment:
Does the Atom spec allow using a date instead of a datetime? This would avoid having to happen wrong information.

Reply:
Thanks for the suggestions @tmattio 😊 
The reason for this is because right now all the dates in changelog .md files are in "YYYY-MM-DD" format, but the Syndic.Date expects ISO 8601 format

Reply:
Atom spec RFC4287 requires a RFC3339 date-time.

https://www.ietf.org/rfc/rfc4287.txt

https://datatracker.ietf.org/doc/html/rfc3339#section-5.6

Reply:
Hi @sabine please I'm a bit confused on what to do?

Reply:
I think you have to do nothing here @oyenuga17. Just confirming to @tmattio that the spec requires a time, so us having to put on here looks fine to me.